### DISCUSSION TITLE
docker-compose vs 'docker compose'

### DISCUSSION THREAD

**Discussion by ssujen** [serverpod/serverpod#610] [Q&A] [answered]
When I install Docker, docker compose V2 is installed.  V2 does not have docker-compose instead it uses 'docker compose'.  I aliased docker-composed to 'docker compose' in .profile. It worked for my Ubuntu but fails on Debian. How/where do I change 'docker-compose' to 'docker compose' ?

**Comment by yahu1031** [comment, marked as answer]
This will be updated soon in the framework. Till then please try managing it as it works for you. 😇

Check on : #560

**Reply by ssujen** [reply]
Thank you for the quick response.  You guys are doing fantastic work!
